src/auth/index.ts

```diff
@@ -71,7 +71,7 @@ export default function createShopifyAuth(options: OAuthStartOptions) {
       ctx.path === inlineOAuthPath ||
       (ctx.path === oAuthStartPath && shouldPerformInlineOAuth(ctx))
     ) {
-      const shop = ctx.query.shop;
+      const shop: string = ctx.query.shop as string;
```

Comment:
Since there's an `as string` here, the type inference shouldn't also need the `: string` declaration

```suggestion
      const shop = ctx.query.shop as string;
```